### PR TITLE
WIP Add mission, vision and scope to OpenZaak

### PR DESCRIPTION
**Pull request by Hugo-ter-Doest**
Fixes #

**Changes**

[Describe the changes here]

### CODE REVIEW

**Comment by sergei-maertens** [comment]
@Hugo-ter-Doest do you have a status update for this PR? How is it progressing?

**Review by joeribekker — APPROVED** [review]
Sounds good

File: OpenZaak_Mission_Vision_Scope.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Mission statement
+What: purpose of OpenZaak
+
+OpenZaak wants to help Dutch municipalities and other government agencies to implement Zaakgericht Werken (ZGW, a Dutch form of case management for municipalities). Moreover it focuses on the registries that are necessary to store the data that is created and needed for ZGW.
+
+Why OpenZaak
+
+We believe that Zaakgericht Werken is a best practice for handling requests from citizens and companies. As such ZGW is a way of working that helps to efficiently organise the backoffice of municipalities on the one hand and to record data in a standardised way that is compliant with laws and regulations.
```
Comment (EdwinCoster) [inline OpenZaak_Mission_Vision_Scope.md:8]: Zaakgericht Werken = ZGW

Comment (EdwinCoster) [inline OpenZaak_Mission_Vision_Scope.md:8]: Als je 'on the one hand' gebruikt, moet je dan ook 'on the other hand' gebruiken?

File: OpenZaak_Mission_Vision_Scope.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+How: OpenZaak proposition
+
+OpenZaak offers an open source software component for recording ZGW data in a standardised way. Data is stored and disclosed through standardised API's following the VNG standard "API's voor Zaakgericht Werken". OpenZaak includes administrative interfaces for IT operatives.
+
+
+# Vision
+We believe that Zaakgericht Werken helps Dutch municipalities to efficiently organise their work and to record and archive their data in a standardised way. The latter is the main concern of OpenZaak but cannot be achieved without a proper implementation of ZGW in the organisation. This is out of scope for OpenZaak.
```
Comment (EdwinCoster) [inline OpenZaak_Mission_Vision_Scope.md:16]: Verander '. This is out of scope' in ', which is out of scope'

File: OpenZaak_Mission_Vision_Scope.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+OpenZaak is open source software. This means that everyone can use OpenZaak to deploy registries for ZGW. Also, everyone can inspect and modify the code of OpenZaak. We encourage this and welcome bug reports, feature requests, implementation reports and the like. OpenZaak will benefit from this, and can be improved.
+
+OpenZaak complies to the "API's for Zaakgericht Werken". Features or changes that require changes to the standard are not added to OpenZaak before the standard is updated. The OpenZaak partners actively contribute bugs and user stories to the standard.
```
Comment (EdwinCoster) [inline OpenZaak_Mission_Vision_Scope.md:22]: Verander 'OpenZaak complies to the "API's for Zaakgericht Werken". Features' in 'As OpenZaak complies to the "API's for Zaakgericht Werken", features'.

**Review by EdwinCoster — CHANGES_REQUESTED** [review]
Ik heb een paar kleine tekstuele aanpassingen voorgesteld.

**Comment by sergei-maertens** [comment]
I'm closing this PR because no activity has happened in the past half-year.

**Pull request closed without merging**